### DISCUSSION TITLE
Is it possible to create/delete content using HTTP POST/DELETE from external website?

### DISCUSSION THREAD

**Discussion by kilianbr** [requarks/wiki#3051] [Help / Questions]
Hey everyone, I really hope I havent overlooked something. I want to build an applicaton where sub systems using an service push to create website content in an distributed system. Is there anyway to create/delete sections of a wiki.js using standard HTTP commands like POST or DELETE? For example is it possible to create a section using a "postman" HTTP command? 

Best regards!

**Comment by mattpr** [comment]
If you look in the docs under developer you see the only documented API is the GraphQL API: 

https://docs.requarks.io/dev/api
https://graphql.org/

So the answer is "yes" there is an API.  Whether that API works the way you are hoping the docs can answer.

This is also an open source project, so you of course can just take the code and implement any API you want that exactly meets your needs.
